src/stream/FFmpegCatchupStream.cpp

```diff
@@ -362,13 +362,6 @@ bool FFmpegCatchupStream::GetTimes(kodi::addon::InputstreamTimes& times)
   return true;
 }
 
-void FFmpegCatchupStream::UpdateCurrentPTS()
-{
-  FFmpegStream::UpdateCurrentPTS();
-  if (m_currentPts != STREAM_NOPTS_VALUE)
-    m_currentPts += m_seekOffset;
-}
-
```

Comment:
This is what we need a workaround for. We need a way of setting m_currentPts for catchup streams.